src/ui/MessageDialog.vala

```diff
@@ -33,7 +33,8 @@ namespace Ev3devKit.Ui {
     public class MessageDialog : Ev3devKit.Ui.Dialog {
         Scroll vscroll;
         Label title_label;
-        Label message_label;
+
+        public Label message_label { get; construct; }
```

Comment:
I am using the dialog to show the pairing key. It is 6 digits and ideally it would be shown in large letters. In my video it is way too small. On the desktop I'd just use pangoscript on the label, but here it is overkill. If you feel like this usecase is too special, I can also make a copy of the dialog as an inner class.